FastMM4.pas

```diff
@@ -4425,7 +4425,7 @@ function AllocateLargeBlock(ASize: NativeUInt {$ifdef LogLockContention}; var AD
   LLargeUsedBlockSize := (ASize + LargeBlockHeaderSize + LargeBlockGranularity - 1 + BlockHeaderSize)
     and -LargeBlockGranularity;
   {Get the Large block}
-  Result := VirtualAlloc(nil, LLargeUsedBlockSize, MEM_COMMIT or MEM_TOP_DOWN,
+  Result := VirtualAlloc(nil, LLargeUsedBlockSize, MEM_COMMIT {$ifdef AllocateLargeBlocksTopDown} or MEM_TOP_DOWN{$endif},

```

Comment:
There is no entry for ``AllocateLargeBlocksTopDown`` in the ``FastMM4Options.inc`` file, so please include this also in your PR with a description why this define might be necessary.
Maybe near https://github.com/pleriche/FastMM4/blob/bed3021e5e543108b4a81e24107a43c68bd93dcd/FastMM4Options.inc#L178